### ISSUE TITLE
Install this template on IIS

### ISSUE THREAD

**Issue by defaron** [akveo/blur-admin#307]
Is it possible running this template on IIS? Thanks.

**Comment by daniel94104** [comment]
Yes. This is a front-end framework. So you can use this frame work with any back end you want.
